lib/derivation.ml

```diff
@@ -18,7 +18,7 @@ let rec derive context = function
         let line_in_file = line in
         Error Known_error.(FialyzerError (UnboundVariable {filename; line=line_in_file; variable=Var v}))
      end
-  | Tuple exprs ->
+  | Tuple (_line, exprs) ->
```

Comment:
Could you use this line number as line of error message in `None` pattern?

Reply:
OK.

Reply:
Sorry, this is my mistake. It's no problem.